docs/1.x/options.md

```diff
@@ -56,13 +56,44 @@ $relay->setOption(Relay::OPT_CLIENT_INVALIDATIONS, false);
 
 ## `OPT_PHPREDIS_COMPATIBILITY`
 
-Relay is [fully compatible](/docs/1.x/compatibility) with PhpRedis. By default however, Relay will return `null` when a key doesn't exist, whereas PhpRedis returns `false`.
+Out of the box Relay is [fully compatible](/docs/1.x/compatibility) with PhpRedis. However, you may disable the compatibility mode, which will cause Relay to:
+
+1. Return `null` when a key doesn't exist, instead of returning `false`
+2. Throw exceptions when a read-error occurs, instead of returning `false`
+3. Not modify `rawCommand()` responses
```

Comment:
`TYPE()` will just return 'none', 'string', 'list', 'hash' etc… instead of the integer defined constants